ompi/mca/coll/sm/coll_sm_module.c

```diff
@@ -507,7 +507,7 @@ static int bootstrap_comm(ompi_communicator_t *comm,
     /* Make the rendezvous filename for this communicators shmem data
        segment.  The CID is not guaranteed to be unique among all
        procs on this node, so also pair it with the PID of the proc
-       with the lowest ORTE name to form a unique filename. */
+       with the lowest PRTE name to form a unique filename. */
```

Comment:
This should be PMIx, not PRTE, I'm 99.99% sure.

Reply:
Done